### PR TITLE
Fix legends

### PR DESCRIPTION
**Pull request by cosa65**
# Background
#### Link to issue 
https://biomage.atlassian.net/browse/BIOMAGE-540

#### Link to staging deployment URL 
https://ui-martinfosco-ui175.scp-staging.biomage.net/

#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [x] Tested locally with Inframock
- [x] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

File: src/utils/plotSpecs/generateFrequencySpec.js [hunk 1]
```diff
@@ -63,7 +71,7 @@ const generateSpec = (config, plotData) => {
         range: 'height',
         nice: true,
         zero: true,
-        domain: { data: 'plotData', field: 'y1' },
+        domain: [0, 100],
```
Comment (StefanBabukov) [inline src/utils/plotSpecs/generateFrequencySpec.js]: Does this line change not break the plot when you switch to count in plots and tables plot type control? The y axis should not be max 100 in this case.

**Review by StefanBabukov — CHANGES_REQUESTED** [review]

**Review by StefanBabukov — APPROVED** [review]

**Comment by vickymorrison** [comment]
Why are the cell set labels different in the categorical embedding and the frequency plot? In the frequency plot they match the cluster annotation from Data Exploration whereas in the categorial embedding they have a weird format that seems to be independent of the cluster annotation from Data Exploration (in other words, I renamed a cluster in data exploration and only the frequency plot updated, not the categorical embedding).

**Comment by cosa65** [comment]
> Why are the cell set labels different in the categorical embedding and the frequency plot? In the frequency plot they match the cluster annotation from Data Exploration whereas in the categorial embedding they have a weird format that seems to be independent of the cluster annotation from Data Exploration (in other words, I renamed a cluster in data exploration and only the frequency plot updated, not the categorical embedding).

The embedding is using internal ids instead of the names, I'll fix that and let you know

**Comment by vickymorrison** [comment]
Very minor point - presumably it should say 'toggle labels' rather than 'toggle legend' here:
Screenshot 2021-03-31 at 20 09 32

**Review by vickymorrison — APPROVED** [review]

**Pull request merged**